bootstrap/src/test/java/io/airlift/bootstrap/TestLifeCycleManager.java

```diff
@@ -41,12 +42,12 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.fail;
 
-@Test(singleThreaded = true)
```

Comment:
@Test(singleThreaded = true) ->
@TestInstance(PER_CLASS) @Execution(SAME_THREAD)
https://github.com/junit-team/junit5/issues/3849

Reply:
Well. It worked so...

Reply:
Yeah, it mostly other way around, for methods which where not annotated `@Test(singleThreaded = true)` you have to ensure that they run concurrently.

Reply:
Added